crates/schema/src/auto_migrate.rs

```diff
@@ -218,6 +224,7 @@ fn auto_migrate_tables(plan: &mut AutoMigratePlan<'_>) -> Result<()> {
                     plan.steps.push(AutoMigrateStep::AddTable(new.key()));
                     Ok(())
                 }
+                // TODO: When we remove tables, we should also remove their dependencies, including row-level security.
```

Comment:
Is this TODO still applicable?

Reply:
Yes, is for the future, until we have triggers or `CASCADE` support